lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -112,10 +133,27 @@ def self.available_options
           FastlaneCore::ConfigItem.new(
             key: :skip_if_exists,
             description: 'If the file already exists in the S3 bucket, skip the upload (and report it in the logs), instead of failing with `user_error!`',
+            deprecated: 'Use if_exists instead',
```

Comment:
I originally modified the `description` field, but that's unnecessary now that we have `deprecated`.

![image](https://github.com/wordpress-mobile/release-toolkit/assets/1218433/5adbfe62-659e-4fed-94a5-1f72af370d1d)